### DISCUSSION TITLE
Cant imagine playing without this incredible addon

### DISCUSSION THREAD

**Discussion by Trae132** [HunterPie/HunterPie#561] [General]
Will you be adapting this to Wilds, and in the past how long does it usually take to release a working version for a new release?  I absolutely love this overlay, so well made.  They should just put you on the payroll.

**Comment by haato3o** [comment]
Definitely. The time taken to add support really depends on the game though, last time MHRise came out, it took me around a week to add basic support to it (monster health, map location and buffs/debuffs tracking), I assume wilds might be similar to MHR since they're made in the same engine, but you never know
